src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/EventHandling/RecordEventsAndPublishToBusOnFirstCallEventListener.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Copyright 2014 SURFnet bv
+ * Copyright 2016 SURFnet B.V.
```

Comment:
RE: edit:

as long as they're still included (Which is the project default) we should keep the hook as is. When it is decided that it is no longer needed, we can remove the headers and hook.